templates/cloud/cloud-computing.html

```diff
@@ -0,0 +1,494 @@
+        <p><a href="/cloud/public-cloud">More about public clouds&nbsp;&rsaquo;</a></p>
+      </div>
+    </div>
+    <div class="col-4">
+      <div class="u-sv1 u-hide--small">
+        {{
+          image (
+          url="https://assets.ubuntu.com/v1/a62acdbe-illustration-private-cloud.svg",
+          alt="Private Cloud",
+          width="98",
+          height="64",
+          hi_def=True,
+          loading="auto"
+          ) | safe
+        }}
+        <h4>Private cloud</h4>
+        <p>A private cloud is owned by an organisation or an individual. All resources are exclusively dedicated to a single entity or a service. It runs on the organisation's premises or in an external data centre. It is managed by the organisation's operations team or a <a href="/blog/what-is-a-managed-it-service">managed service provider</a>.</p>
+        <p><a href="/cloud/private-cloud">More about private clouds&nbsp;&rsaquo;</a></p>
+      </div>
+    </div>
+    <div class="col-4">
+      <div class="u-sv1 u-hide--small">
+        {{
+          image (
+          url="https://assets.ubuntu.com/v1/3b176ef1-managed-private-cloud.svg",
+          alt="",
+          width="64",
+          height="64",
+          hi_def=True,
+          loading="lazy"
+          ) | safe
+        }}
+        <h4>Managed cloud</h4>
+        <p>Managed clouds are private clouds that are fully managed by a third-party organisation (aka managed service provider). The customer provides the hardware, but cloud operations and maintenance tasks are outsourced. The cloud can either run on the organisation's premises or in the managed service provider's data centre.</p>
+        <p><a href="/managed">More about managed clouds&nbsp;&rsaquo;</a></p>
+      </div>
+    </div>
+    </div>
+    <div class="row">
+      <div class="col-4">
+        <div class="u-sv1 u-hide--small">
+          {{
+            image (
+            url="https://assets.ubuntu.com/v1/2dc4373b-illustration-edge-cloud.svg",
+            alt="",
+            width="94",
+            height="64",
+            hi_def=True,
+            loading="lazy"
+            ) | safe
+          }}
+          <h4>Micro cloud</h4>
+          <p>Micro clouds are a new class of infrastructure for on-demand computing at the edge. They differ from the internet-of-things (IoT), which uses thousands of single machines or sensors to gather data, yet they perform computing tasks. Micro clouds reuse proven cloud primitives but with the unattended, autonomous and clustering features that resolve typical edge computing challenges.</p>
+          <p><a href="/edge-computing">More about micro clouds&nbsp;&rsaquo;</a></p>
+        </div>
+      </div>
+      <div class="col-4">
+        <div class="u-sv1 u-hide--small">
+          {{
+            image (
+            url="https://assets.ubuntu.com/v1/45120edb-Hybrid+Cloud.svg",
+            alt="Hybrid Cloud",
```

Comment:
Should probably do alt text on all or none for these little logos. i'd personally say none

Reply:
I removed all `alt` text, I think in this case, they don't represent anything, at least in the logos they represent a brand or company.